### PR TITLE
[WIP] Make edge ids optional

### PR DESCRIPTION
**Pull request by VibhuJawa**
This PR makes edge ids optional for cugraph-dgl dataloaders

Todo: 
- [ ] Add tests

### CODE REVIEW

File: python/cugraph-dgl/cugraph_dgl/nn/conv/transformerconv.py [hunk 1]
```diff
@@ -15,8 +15,8 @@
 from cugraph_dgl.nn.conv.base import BaseConv
 from cugraph.utilities.utils import import_optional
```
Comment (VibhuJawa) [inline python/cugraph-dgl/cugraph_dgl/nn/conv/transformerconv.py:17]: Remove after https://github.com/rapidsai/cugraph/pull/3693 merges

File: python/cugraph-dgl/cugraph_dgl/nn/conv/gatconv.py [hunk 2]
```diff
@@ -19,8 +19,8 @@
 from cugraph_dgl.nn.conv.base import BaseConv
 from cugraph.utilities.utils import import_optional
```
Comment (VibhuJawa) [inline python/cugraph-dgl/cugraph_dgl/nn/conv/gatconv.py:21]: Remove after https://github.com/rapidsai/cugraph/pull/3693 merges

**Review by alexbarghi-nv — APPROVED** [review]
👍

**Pull request merged**